packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -1166,6 +1167,13 @@ mixin SchedulerBinding on BindingBase {
       return true;
     }());
   }
+
+  /// When a scheduling error occurs, resets the relative state of EventLoop.
+  @visibleForTesting
+  void resetEventLoop() {
```

Comment:
Call this debugResetEventLoop to make it clear that this shouldn't be called by an app and make it only do something when asserts are enabled.

Reply:
Yeah I'll try to change it

Reply:
done you can review it